guides/v2.1/install-gde/prereq/nginx.md

```diff
@@ -75,57 +75,76 @@ Refer to [MySQL]({{ page.baseurl }}/install-gde/prereq/mysql.html) for more info
 
 There are several ways to download the Magento software, including:
 
-*	[Download an archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html)
-
 *	[Get the Composer metapackage]({{ page.baseurl }}/install-gde/composer.html)
 
-*	[Clone the git repository]({{ page.baseurl }}/install-gde/prereq/dev_install.html)
-
-For this example, we'll download and extract an archive.
+*	[Download an archive]({{ page.baseurl }}/install-gde/prereq/zip_install.html)
 
-2. Change to the web server docroot directory, or to a directory you’ve configured as a virtual host docroot. For this example, we're using the Ubuntu default `/var/www/html`.
+*	[Clone the git repository]({{ page.baseurl }}/install-gde/prereq/dev_install.html)
 
-		cd /var/www/html
+For this example, we'll install using Composer and the Command Line.
```

Comment:
"Command Line" shouldn't be capitalized. See line 87.

Reply:
Fixed and pushed